source/text/Font.h

```diff
@@ -72,15 +72,18 @@ class Font {
 	std::string TruncateFront(const std::string &str, int &width) const;
 	std::string TruncateMiddle(const std::string &str, int &width) const;
 
+private:
+	/// Shared VAO and VBO quad (0,0) -> (1,1)
+	static GLuint vao;
+	static GLuint vbo;
 
```

Comment:
Why is this not in the anonymous namespace? Actually, the only reason the rest of the uniform locations, sizes, etc wasn't in the anonymous namespace was because we initialised everything separately, we should follow the standard of the other shaders and move those there too.